app/src/main/java/com/yterletskyi/happyfriend/features/pin/domain/SingleLiveEvent.java

```diff
@@ -0,0 +1,47 @@
+package com.yterletskyi.happyfriend.features.pin.domain;
```

Comment:
can we please move it to some place in `common` package - because we are going to use this class all over the project